### PR TITLE
Fix spirv path in bruteforce tests

### PR DESCRIPTION
**Pull request by gwawiork**
Fix issue #355

### CODE REVIEW

File: test_conformance/math_brute_force/main.c [hunk 1]
```diff
@@ -1843,26 +1810,13 @@ int MakeKernels( const char **c, cl_uint count, const char *name, cl_uint kernel
       strcat(options, " -cl-fast-relaxed-math");
     }
 
-    *p = clCreateProgramWithSource( gContext, count, c, NULL, &error );
-    if( NULL == *p || error )
+    error = create_single_kernel_helper(gContext, p, NULL, count, c, NULL, options);
+    if ( error != CL_SUCCESS )
     {
         vlog_error( "\t\tFAILED -- Failed to create program. (%d)\n", error );
         return -1;
```
Comment (kpet) [inline test_conformance/math_brute_force/main.c]: Should this be changed to `return error` as well?

**Comment by gwawiork** [comment]
In my opinion not, offline spirv compilation is available since CL2.1

**Comment by kpet** [comment]
Sure, but there is nothing SPIR-V specific that I could spot in the patch. If the patch applies I think we should port it, if anything to reduce the number of places where the code sequence for building a program exists.

**Comment by gwawiork** [comment]
Ok I will prepare those fixes.
Additionally I've noticed more places in main.c where clCreateProgramWithSource has to be exchanged with create_single_kernel_helper. Fix in progress.

**Comment by gwawiork** [comment]
All fixes have been pushed out

**Comment by kpet** [comment]
Thanks! Merging all PRs.

**Pull request merged**